tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py27,py34}-django{15,16,17,18,19,master}, py33-django{15,16,17,18}, py26-django{15,16}, lint, docs
+envlist = {py27,py34}-django{15,16,17,18,19,110,master}, py33-django{15,16,17,18}, py26-django{15,16}, lint, docs
```

Comment:
Ah yes, silly me!

On 6 Sep 2016 16:10, "Aarni Koskela" <notifications@github.com> wrote:

> In tox.ini
> <https://github.com/python-babel/django-babel/pull/33#discussion_r77654427>
> :
>
> > @@ -1,5 +1,5 @@
> >  [tox]
> > -envlist = {py27,py34}-django{15,16,17,18,19,master}, py33-django{15,16,17,18}, py26-django{15,16}, lint, docs
> > +envlist = {py27,py34}-django{15,16,17,18,19,110,master}, py33-django{15,16,17,18}, py26-django{15,16}, lint, docs
>
> What would be a better way to describe Django 1.10?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/python-babel/django-babel/pull/33/files/f1e93e370d3376a1a8b959fe95a89d3ddc8fc832#r77654427>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAZQTDWm_Kkj89tR5pusgRFZfpxYaky0ks5qnYJggaJpZM4J18Di>
> .
>